### PR TITLE
fix: alignment of text inside the child element to that of Exapandable ListView

### LINKED ISSUE #851: Exapndable List View in Faq Activity are not align with the child elements
**Summary:**

In Faq Activity, the Expandable ListView is not aligned with the child view text.

**Steps to reproduce:**

Navigate to FAQ section clicking on the overlay menu icon.expand the list by clicking on down arrow. 

**Expected behaviour:**

The child view after the expanding of ListView should be the same align to each other.

**Observed behaviour:**

Currently, the child views text that shows the answer is not aligned to the text of the ListView.

**Device and Android version:**

Android Version - Android 10
Android Device - POCO X2

**Screenshots:**

Screenshot_2020-03-23-12-37-08-333_org mifos mobilewallet mifospay

### PR DESCRIPTION
**Pull request by refactor-droidyy**
## Issue Fix
Fixes #851 

## Screenshots

Screenshot_2020-03-23-12-03-38-569_org mifos mobilewallet mifospay (1)

## Description

Adjust Margin 

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/res/layout/faq_list_child.xml [hunk 1]
```diff
@@ -8,7 +8,10 @@
         android:id="@+id/faq_list_child"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_margin="16dp"
+        android:layout_marginLeft="40dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginBottom="16dp"
```
Comment (jawidMuhammadi) [inline mifospay/src/main/res/layout/faq_list_child.xml]: Hey, @refactor-droidyy  your change looks great. But it would be much better if you avoid using hardcoded values, instead use the predefined values from `dimens.xml` file or add a new one if it is not there

Comment (refactor-droidyy) [reply mifospay/src/main/res/layout/faq_list_child.xml]: Ya, sure I will soon update this PR.

Comment (refactor-droidyy) [reply mifospay/src/main/res/layout/faq_list_child.xml]: Ya, sure I will soon update this PR.

**Comment by refactor-droidyy** [comment]
@jawidMuhammadi Done :+1:

**Comment by refactor-droidyy** [comment]
@luckyman20, please review

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<img height='175' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
+<img height='175' src="https://user-images.githubusercontent.com/44283521/78983673-455cf780-7b42-11ea-849e-ecd2009dd562.png" align="left" hspace="1" vspace="1">
```
Comment (luckyman20) [inline README.md:1]: Is this related to this issue?
